pkg/utils/kubeclient/volume.go

```diff
@@ -196,6 +196,30 @@ func GetPvcMountPods(e client.Client, pvcName, namespace string) ([]v1.Pod, erro
 	return pods, err
 }
 
+// GetPvcMountNodes get nodes which have pods mounted the specific pvc for a given namespace
+// it will only return a map of nodeName and amount of PvcMountPods on it
+func GetPvcMountNodes(e client.Client, pvcName, namespace string) (map[string]int64, error) {
+	pvcMountNodes := map[string]int64{}
+	pvcMountPods, err := GetPvcMountPods(e, pvcName, namespace)
+	if err != nil {
+		log.Error(err, "Failed to get PVC Mount Nodes because cannot list pods")
+		return pvcMountNodes, err
+	}
+
+	for _, pod := range pvcMountPods {
+		nodeName := pod.Spec.NodeName
+		if nodeName == "" {
+			continue
+		}
+		if _, found := pvcMountNodes[nodeName]; !found {
+			pvcMountNodes[nodeName] = 1
```

Comment:
why not found also have key for nodeName and value is 1?

Reply:
I think it's kind of bitmap. I prefer   map[string]bool